backend/.php_cs

```diff
@@ -14,7 +14,8 @@ return PhpCsFixer\Config::create()
         '@DoctrineAnnotation' => true,
         '@PhpCsFixer' => true,
         'braces' => ['position_after_functions_and_oop_constructs' => 'same'],
-        'php_unit_test_class_requires_covers' => false
+        'php_unit_test_class_requires_covers' => false,
+        'no_superfluous_phpdoc_tags' => true,
```

Comment:
will you revert the rule in this PR in a follow up commit, or do we merge it?

Reply:
This one I would merge, because it is not considered risky. If we remove it, cs-fixer will start complaining about now incomplete phpdoc annotations I think.